src/entity/Project.java

```diff
@@ -1,2 +1,12 @@
-package entity;public class Project {
+package entity;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public interface Project {
+
+    String getProjectTitle();
+
+    ArrayList<HashMap<String, String>> getTasks();
```

Comment:
ISSUE: please update this to work with Task entity, since we are not using Hashmap<String, String> to represent tasks anymore

Reply:
This was a mock entity that was never used. It was committed by accident, I will remove it.